packages/melody-streams/src/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2018 trivago N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { createComponent } from './component';
+export { createState } from './operators/createState';
+export { mergeObject } from './operators/mergeObject';
```

Comment:
I'd argue that `mergeObject` and `mergeIntoObject` should not be part of the public API. `combine` can do both their jobs and its easier to learn just one function instead of 3.